app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb

```diff
@@ -27,17 +27,22 @@ def process_alert(operation, miq_alert)
 
     def convert_to_group_trigger(miq_alert)
       eval_method = miq_alert[:conditions][:eval_method]
-      firing_match = case eval_method
-                     when "mw_heap_used", "mw_non_heap_used" then 'ANY'
-                     else 'ALL'
-                     end
+      firing_match = 'ALL'
+      context = { 'dataId.type' => 'gauge', 'dataId.prefix' => 'hm_g_' }
```

Comment:
Note that adding this context is purely informative. 
It has not any logic behind that but I think is good to put this info on the definitions.
Context can be used for these tasks nicely.